website/docs/r/one_dashboard.html.markdown

```diff
@@ -357,7 +368,95 @@ Each widget type supports an additional set of arguments:
     * `initial_sorting` - (Optional) An attribute that describes the sorting mechanism for the table. This attribute requires specifying the following attributes in a nested block -
         * `name` - (Required) The name of column to be sorted. Examples of few valid values are `timestamp`, `appId`, `appName`, etc.
         * `direction` - (Required) Defines the sort order. Accepted values are `asc` for ascending or `desc` for descending.
+    * `data_format` - (Optional) A nested block that describes data format. See [Nested data_format blocks](#nested-data_format-blocks) below for details.
+        
+
+### Nested `data_format` blocks
+
+Nested `data_format` blocks help specify the format of data displayed by a widget per attribute in the data returned by the NRQL query rendering the widget; thereby defining how the data fetched is best interpreted. This is supported for **billboards** and **tables**, as these are the only widgets in dashboards which return single or multi-faceted data that may be formatted based on the type of data returned.
+This attribute requires specifying the following attributes in a nested block -
+
+  * `name` - (Required) This attribute mandates the specification of the column name to be formatted. It identifies which column the data format should be applied to.
+  * `type` - (Required) This attribute sets the format category for your data. Accepted values include - 
+    - `decimal` for numeric values
+    - `date` for date/time values
+    - `duration` for length of time
+    - `recent-relative` for values referencing a relative point in time
+    - `custom` to be used with date/time values, in order to select a specific format the date/time value would need to be rendered as
+    - `humanized` to be used with numeric values, in order to enable Autoformat
+  * `format` - (Optional) This attribute is provided when the `name` is that of a column comprising date/time values and the `type` attribute is set to `custom` defining the specific date format to be applied to your data.
+
+  |     Accepted value  |        Format           |                     
+  |---------------------|-------------------------| 
+  | `%b %d, %Y`         | `MMM DD,YYYY`           | 
+  | `%d/%m/%Y`          | `DD/MM/YYYY(EU)`        | 
+  | `%x`                | `DD/MM/YYYY(USA)`       | 
+  | `%I:%M%p`           | `12-hour format`        | 
+  | `%H:%Mh`            | `24-hour format`        | 
+  | `%H:%Mh UTC (%Z)`   | `24-hour with timezone` | 
+  | `%Y-%m-%dT%X.%L%Z`  | `ISO with timezone`     | 
+  | `%b %d, %Y, %X`     | `MMM DD, YYYY,hh:mm:ss` | 
+  | `%X`                | `hh:mm:ss`              | 
+
+* `precision` - (Optional) This attribute is utilized when the `type` attribute is set to `decimal`, stipulating the precise number of digits after the decimal point for your data.
```

Comment:
we might need to unindent the point with `precision`

<img width="640" alt="image" src="https://github.com/user-attachments/assets/97a5763f-84ae-4474-93ab-a657f28b15f8">